pkgs/top-level/all-packages.nix

```diff
@@ -8589,8 +8589,8 @@ in
   };
 
   ccls = callPackage ../development/tools/misc/ccls {
-    llvmPackages = llvmPackages_latest;
-    stdenv = llvmPackages_latest.stdenv;
```

Comment:
This should be llvm_8 right now, so this is a downgrade to llvm_7.

Reply:
So maybe the actual problem is, that it does not work with llvm 8.

Reply:
Ah. That might actually be it. Maybe the version 5 thing was me misreading build output.